sdk/servicebus/azure-servicebus/azure/servicebus/management/_generated/aio/_service_bus_management_client_async.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Any, Optional
+
+from azure.core import AsyncPipelineClient
+from msrest import Deserializer, Serializer
+
+from ._configuration_async import ServiceBusManagementClientConfiguration
+from .operations_async import QueueOperations
+from .operations_async import ServiceBusManagementClientOperationsMixin
+from .. import models
+
+
+class ServiceBusManagementClient(ServiceBusManagementClientOperationsMixin):
+    """Azure Service Bus client for managing Queues, Topics, and Subscriptions.
+
+    :ivar queue: QueueOperations operations
+    :vartype queue: azure.service._control_client2.aio.operations_async.QueueOperations
+    :param endpoint: The Service Bus fully qualified domain name.
+    :type endpoint: str
+    :keyword int polling_interval: Default waiting time between two polls for LRO operations if no Retry-After header is present.
```

Comment:
I'm just ask some stupid question for this "polling_interval": what is LRO operation? and where and how can user set Retry-After header?